continuous_integration/travis/install.sh

```diff
@@ -61,14 +61,7 @@ install_conda()
 install_pip()
 {
     # Install PyQt5
-    conda install 'pyqt>=5.6.0';
-
-    # Install testing packages
-    pip install pytest pytest-cov pytest-qt mock
-
-    # Install extra packages
-    EXTRA_PACKAGES="matplotlib pandas sympy pyzmq pillow"
-    pip install $EXTRA_PACKAGES
```

Comment:
I only revert the installation of pyqt (to install it with conda), the other dependencies will be installed by `pip install -e .[test]`